packages/itwinui-react/src/core/Tile/TileNew.tsx

```diff
@@ -0,0 +1,177 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import React from 'react';
+import cx from 'classnames';
+import {
+  // StatusIconMap,
+  // useTheme,
+  // SvgMore,
+  // SvgNew,
+  // SvgCheckmark,
+  // LinkAction,
+  PolymorphicComponentProps,
+  // useSafeContext,
+  // supportsHas,
+  PolymorphicForwardRefComponent,
+} from '../utils';
+import '@itwin/itwinui-css/css/tile.css';
+// import { DropdownMenu } from '../DropdownMenu';
+// import { IconButton } from '../Buttons';
+// import { ProgressRadial } from '../ProgressIndicators';
+// import Tile from '.';
+
+//<Tile>
+//  <Tile.Thumbnail>
+//  <Tile.Name>
+//  <Tile.Content>  //has meta data
+//  <Tile.Buttons>
+//</Tile>
+
+// ----------------------------------------------------------------------------
+// Main Tile component
+
+const TileComponent = React.forwardRef((props, ref) => {
+  const { as: Element = 'div', className, status, ...rest } = props;
+  return (
+    <Element
+      className={cx(
+        'iui-tile',
+        {
+          //   'iui-folder': variant === 'folder',
+          //   'iui-new': isNew,
+          //   'iui-selected': isSelected,
+          //   'iui-actionable': isActionable,
+          [`iui-${status}`]: !!status,
+          //   'iui-loading': isLoading,
+        },
+        className,
+      )}
+      ref={ref}
+      {...rest}
+    />
+  );
+}) as PolymorphicForwardRefComponent<'div', TileOwnProps>;
+
+type TileOwnProps = {
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
```

Comment:
The main Tile needs to have all the high-level state props, including (in addition to `status`): `isSelected`, `isNew`, `variant`, `isActionable`, `isLoading` and `isDisabled`

Reply:
Added these